src/test/java/com/ibm/cloud/sdk/core/test/security/ContainerAuthenticatorTest.java

```diff
@@ -65,13 +69,29 @@
   private static final String mockScope = "scope1";
   private static final String mockCRToken = "cr-token-1";
 
+  // Logging level used by this test.
+  // For debugging, set this to Level.FINE or Level.ALL, etc.
+  private static Level logLevel = Level.SEVERE;
+
   @Override
   @BeforeMethod
   public void setUp() throws Exception {
     super.setUp();
     url = getMockWebServerUrl();
     tokenData1 = loadFixture("src/test/resources/iam_token.json", IamToken.class);
     tokenData2 = loadFixture("src/test/resources/refreshed_iam_token.json", IamToken.class);
+
+    // Set up java.util.logging to display messages on the console.
+    ConsoleHandler handler = new ConsoleHandler();
+    handler.setLevel(logLevel);
+    Logger logger;
+    logger = Logger.getLogger(ContainerAuthenticator.class.getName());
+    logger.setLevel(logLevel);
+    logger.addHandler(handler);
+
+    logger = Logger.getLogger(TokenRequestBasedAuthenticator.class.getName());
+    logger.setLevel(logLevel);
+    logger.addHandler(handler);
```

Comment:
Why not
```java
    Logger logger = Logger.getLogger(ContainerAuthenticator.class.getName());
```
just like below?
Also, doesn't the second logger overrides the first one?